selfdrive/controls/controlsd.py

```diff
@@ -576,8 +576,10 @@ def state_control(self, CS):
       actuators.steer, actuators.steeringAngleDeg, lac_log = self.LaC.update(CC.latActive, CS, self.VM, params,
                                                                              self.last_actuators, desired_curvature,
                                                                              desired_curvature_rate, self.sm['liveLocationKalman'])
+      controlsState = log.ControlsState.new_message(desiredCurvature=desired_curvature, desiredCurvatureRate=desired_curvature_rate)
```

Comment:
is this convention okay? I notice we hardly use it (if at all). we can also do `log.ControlsState(desiredCurvature=...)`